Dockerfile

```diff
@@ -3,8 +3,9 @@ FROM php:8-cli-alpine
 WORKDIR /etc/rector
 
 # required for composer patches
-RUN apk add --no-cache patch
+RUN apk add --no-cache patch git
 
 COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
 
 RUN mkdir -p /etc/rector
+RUN git config --global --add safe.directory /etc/rector
```

Comment:
These changes make it possible to run `composer complete-check` from within the docker container.

Reply:
Could you create separate PR for it?

Reply:
Good call. #5987